core/src/mill/define/Module.scala

```diff
@@ -82,12 +85,13 @@ object Module{
     // script/REPL runner always wraps user code in a wrapper object/trait
     def reflectNestedObjects[T: ClassTag] = {
       (reflect[T] ++
-        outer
-          .getClass
-          .getClasses
+        outer.getClass.getClasses
           .filter(implicitly[ClassTag[T]].runtimeClass isAssignableFrom _)
-          .flatMap(c => c.getFields.find(_.getName == "MODULE$").map(_.get(c).asInstanceOf[T]))
```

Comment:
Perhaps we could allow this to remain on one line by configuring the max line length to 100?